pkg/types/jsonpatch.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package types
+
+// JSONPatch represents a json patch for an object
+type JSONPatch struct {
+	// Relative file path within the kustomization for a json patch file.
+	Path string `json:"path" yaml:"path"`
+
+	// Target refers to a Kubernetes object that the json patch will be
```

Comment:
`PatchTarget`

Or - just leave is as `Target` and fix the name of the field below to be just `Target` (which is OK if it is declared in a `patching` package).

Reply:
`Target` sounds better since it's already in the patch package. We can also embed `Target`  in `PatchJson6902`.  How does that look? The format is:
```
patchJson6902:
- path: patch.JSON
  version: v1
  kind: deployment
  name: some deployment
```